typescript/appconfig-hosted-configuration-lambda-extension/src/lambda-handler.ts

```diff
@@ -0,0 +1,67 @@
+import * as http from 'http';
+
+
+export const handler = async (): Promise<any> => {
+	// retrieve AppConfig configuration data from Lambda extension
+	const res: any = await new Promise((resolve) => {
+		http.get(
+			`http://localhost:2772/applications/AppConfigSampleApplication/environments/AppConfigSampleLambdaDevelopmentEnvironment/configurations/AppConfigSampleConfigurationProfile`,
```

Comment:
Maybe I'm missing something, but I dont see where this URL is coming from

Reply:
I believe this only works with Lambda/ECS/Fargate workloads, normally you would acquire these config values via the AWS SDK: https://docs.aws.amazon.com/appconfig/latest/userguide/appconfig-integration-lambda-extensions.html#appconfig-integration-lambda-extensions-enabling

There's also this blog post that references the localhost URL proxy: https://aws.amazon.com/blogs/mt/introducing-aws-appconfig-lambda-extension-deploying-application-configuration-serverless/